application/classes/beans/setup/update/v/1/4/1.php

```diff
@@ -0,0 +1,35 @@
+<?php defined('SYSPATH') or die('No direct script access.');
+/*
+BeansBooks
+Copyright (C) System76, Inc.
+
+This file is part of BeansBooks.
+
+BeansBooks is free software; you can redistribute it and/or modify
+it under the terms of the BeansBooks Public License.
+
+BeansBooks is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+See the BeansBooks Public License for more details.
+
+You should have received a copy of the BeansBooks Public License
+along with BeansBooks; if not, email info@beansbooks.com.	
+*/
+
+class Beans_Setup_Update_V_1_4_1 extends Beans_Setup_Update_V {
+
+	public function __construct($data = NULL)
+	{
+		parent::__construct($data);
+	}
+	
+	protected function _execute()
+	{
+		$this->_db_update_table_column(	'form_lines', 'quantity', 'decimal( 13, 3 ) DEFAULT NULL');
```

Comment:
This is close, it just has to reference the field name again and specify "NULL DEFAULT NULL" - I'm basically copying what was done here without changing the column name: 
https://github.com/system76/beansbooks/blob/master/application/classes/beans/setup/update/v/1/3.php#L36

```
$this->_db_update_table_column(	'form_lines', 'quantity', '`quantity` decimal( 13, 3 ) NULL DEFAULT NULL');
```